README.md

```diff
@@ -82,4 +82,4 @@ or
 
 ##Abstraction
 
-Privacy Cookie Bundle v0.2 allows to increase abstraction level on the implementation side. Example of usage is shown on [EzDemoBundle](https://github.com/ezsystems/DemoBundle) (part of the eZ Publish / eZ Platform) in which banner options are retrieved from specified content type structure.
+Privacy Cookie Bundle v0.2 allows to increase abstraction level on the implementation side. Example of the usage can be see at [EzDemoBundle](https://github.com/ezsystems/DemoBundle). Banner options are retrieved from a specified content type structure.
```

Comment:
+Privacy Cookie Bundle v0.2 allows increasing the abstraction level on the implementation side. Example of the usage can be see at [EzDemoBundle](https://github.com/ezsystems/DemoBundle). Banner options are retrieved from a specified Content Type structure.